marketplace-service/src/main/java/com/axonivy/market/service/impl/ProductServiceImpl.java

```diff
@@ -430,6 +433,32 @@ private void mappingLogoFromGHContent(Product product, GHContent ghContent) {
     }
   }
 
+  private void mappingVendorImageFromGHContent(Product product, GHContent ghContent) {
+    if (StringUtils.endsWith(ghContent.getName(), MetaConstants.META_FILE)) {
+      mapVendorImage(product, ghContent, product.getVendorImage(), false);
+      mapVendorImage(product, ghContent, product.getVendorImageDarkMode(), true);
+    }
+  }
+
+  private void mapVendorImage(Product product, GHContent ghContent, String imageName, boolean isDarkMode) {
```

Comment:
Yes, because in meta.json, I introduce another property for vendor dark mode, so when mapping to Product document, it has to know which is dark mode and which is light mode vendor image